darwin/future/data_objects/release.py

```diff
@@ -19,6 +19,9 @@ class Release(DefaultDarwin):
 
     name: str
 
+    def __str__(self) -> str:
```

Comment:
Is there a purpose to overwriting this? Pydantic objects already print to console?